topics/single-cell/tutorials/scrna-seurat-pbmc3k/tutorial.md

```diff
@@ -0,0 +1,1499 @@
+
+{% include _includes/cyoa-choices.html option1='Separate Preprocessing Steps' option2='SCTransform' defualt='Separate-Preprocessing-Steps' text="You can perform each preprocessing step separately, which might give you a better understanding of the different elements involved in preprocessing, or run them all at once using SCTransform." %}
+
+<div class='Separate-Preprocessing-Steps' markdown='1'>
+><hands-on-title>Separate Preprocessing Steps</hands-on-title>
+>
+> 1. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `Filtered Dataset` (output of **Seurat Create** {% icon tool %})
+>    - *"Method used"*: `Normalize with 'NormalizeData'`
+>        - *"Method for normalization"*: `LogNormalize`
+>
+> 2. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Identify highly variable genes with 'FindVariableFeatures'`
+>        - *"Method to select variable features"*: `vst`
+>        - *"Output list of most variable features"*: `Yes`
+>
+> 3. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Scale and regress with 'ScaleData'`
+>        - *"Regress out a variable"*: `No`
+>        - *"Features to scale"*: `All Features`
+>
+> 4. Rename the output as `Preprocessed Data`
+>
+{: .hands_on}
+</div>
+
+<div class='SCTransform' markdown='1'>
+><hands-on-title>SCTransform</hands-on-title>
+>
+> 1. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `Filtered Dataset` (output of **Seurat Create** {% icon tool %})
+>    - *"Method used"*: `Complete all preprocessing with 'SCTransform'`
+>        - *"Genes to calculate residual features for"*: `all genes`
+>            - *"How to set variable features"*: `set number of variable features`
+>        - *"Output list of most variable features"*: `Yes`
+>        - *"Variable(s) to regress out"*: `percent.mt`
```

Comment:
isn't it different from the normal scaling step? In the normal scaling step, there was no variable regressed out but in scTransform, `percent.mt` regressed out